AppCenterCrashes/AppCenterCrashesTests/MSACWrapperCrashesHelperTests.mm

```diff
@@ -301,4 +301,19 @@ - (void)testTrackExceptionWithAllParameters {
   XCTAssertEqualObjects(actualErrorId, errorId);
 }
 
+- (void)testBuildHandledErrorReportWithErrorID {
```

Comment:
Please add a check for `report.device`, which should be `EqualObjects` to `[[MSACDeviceTracker sharedInstance] device]`.